docs/faq.rst

```diff
@@ -1,14 +1,7 @@
 Frequently asked questions
 ==========================
 
-**Can a user iterate over irregular topology instead of grids, such as tetrahedra meshes, line segment vertices?**
-These structures have to be represented using 1D arrays in Taichi. You can still iterate over it using `for i in x` or `for i in range(n)`.
-However, at compile time, there's little the Taichi compiler can do for you to optimize it. You can still tweak the data layout to get different run time cache behaviors and performance numbers.
+**Q:** Can a user iterate over irregular topology instead of grids, such as tetrahedral meshes, line segment vertices?
```

Comment:
*''such as tetrahedral meshes, line segment vertices?''*
I'm not sure if you mean *line segments, vertices, etc.?*

Reply:
Thanks for pointing this out. I removed *line segment vertices* for clarity.